SwiftyStoreKit/OS.swift

```diff
@@ -35,14 +35,22 @@ import StoreKit
 #endif
 
 // MARK: - Missing SKError on watchOS
-#if os(watchOS)
+#if os(watchOS) && swift(<5.3)
 public struct SKError: Error {
     
-    var Code: SKErrorCode = .unknown
-    var _nsError: NSError?
+    public typealias Code = SKErrorCode
```

Comment:
Is `typealias` available prior to Swift 5.3? I feel like I've used it a few times before this...

Reply:
The reason behind the swift(<5.3) is that apple forgot to add SKError to the watchOS SDK so be have to build our own, they fixed that with with 5.3 oder lets say with the new version of Xcode which ships 5.3